srml/babe/src/lib.rs

```diff
@@ -155,6 +222,18 @@ impl<T: Trait> session::OneSessionHandler<T::AccountId> for Module<T> {
 				Self::change_authorities(next_authorities);
 			}
 		}
+
+		let rho = UnderConstruction::get();
+		UnderConstruction::put([0; 32]);
+		let last_epoch_randomness = EpochRandomness::get();
+		let epoch_index = EpochIndex::get().wrapping_add(1);
```

Comment:
Again use `mutate` and do we support that the epoch starts from `0` again?

Reply:
@bkchr the index is a `u64`, so we shouldn't have to deal with that. But it's also a bit weird to use `wrapping_add` if so.

Reply:
Yeah, that should never overflow, I just was curious because of the `wrapping_add`.